Makefile.pseudomodules

```diff
@@ -23,3 +23,6 @@ PSEUDOMODULES += netif
 # include variants of the AT86RF2xx drivers as pseudo modules
 PSEUDOMODULES += at86rf23%
 PSEUDOMODULES += at86rf21%
+
+# add all pseudo random number generator variants as pseudomodules
+PSEUDOMODULES += prng_%
```

Comment:
Either use the prefix `random_` or rename the module. This module is already a mess as it is (`genrand_` as function prefix vs. `random` as module name), so we don't need more … randomness (badum tzz)